src/nvim/plines.h

```diff
@@ -4,25 +4,88 @@
 #include <stdint.h>  // IWYU pragma: keep
 
 #include "nvim/marktree_defs.h"
+#include "nvim/mbyte_defs.h"
 #include "nvim/pos_defs.h"  // IWYU pragma: keep
 #include "nvim/types_defs.h"
 
-/// Argument for lbr_chartabsize().
+typedef bool CSType;
+
+enum {
+  CHARSIZE_REGULAR = false,
```

Comment:
`kCharsizeRegular` and `kCharsizeFast` and no need for initializers.

Reply:
I added explicit initializers to show that these constants are used as booleans and not integers. Should I still remove them?

Reply:
They are used as enum values, where the exact representation doesn't matter. these just get co-erced to 0/1 anyway. So yes, they should be removed.